process/accounts/accountsProcessor.go

```diff
@@ -24,15 +23,13 @@ var log = logger.GetOrCreate("indexer/process/accounts")
 type accountsProcessor struct {
 	internalMarshalizer    marshal.Marshalizer
```

Comment:
can be removed because is no longer used.

Reply:
good catch. removed